client/src/components/heading.css

```diff
@@ -0,0 +1,27 @@
+
+.button{
+    position: absolute;
+top: 80%;
```

Comment:
We need indentation because it improves redability of the code - if we all follow the same formatting standards it will be easier to focus on important things when reading/reviewing the code (like logic, and implementation).
The CSS needs to be formatted, the indentation looks off. It should look something like:
```
.class-name {
    rule: something;
    rule: something-else;
}

.next-class {
    rule: something;
    rule: something-else;
}
```